### PR TITLE
PP-4462 Bank details submission

### PR DESCRIPTION
**Pull request by DanailMinchev**
## WHAT

This PR contains Stripe Connect bank details submission implementation and minor tweaks (disable autocomplete and error messages).

Please see commits for more information.

with @alan-gds

### CODE REVIEW

File: app/controllers/stripe-setup/bank-details/post.controller.js [hunk 1]
```diff
@@ -18,24 +22,53 @@ module.exports = (req, res) => {
...
+        return res.redirect(303, paths.dashboard.index)
+      })
+      .catch(error => {
+        // check if it is Stripe related error
+        if (error.code) {
+          if (error.code === 'routing_number_invalid') {
+            // invalid sort code
+            errors = {}
+            errors[SORT_CODE_FIELD] = fieldValidationChecks.validationErrors.invalidSortCode
+            pageData.errors = errors
```
Comment (stephencdaly) [inline app/controllers/stripe-setup/bank-details/post.controller.js]: i think this would be clearer if it were
```pageData.errors = { SORT_CODE_FIELD:  fieldValidationChecks.validationErrors.invalidSortCode }```
re-using the errors object makes it a bit hard to follow

Comment (DanailMinchev) [reply app/controllers/stripe-setup/bank-details/post.controller.js]: `SORT_CODE_FIELD` is constant which is evaluated as a property name, that is why we need `errors[SORT_CODE_FIELD]` which will be resolved to `errors { 'account-number': ... }`
I changed the controller similar way: `pageData.errors[SORT_CODE_FIELD] = ...` and made errors `const` again, thanks

Comment (stephencdaly) [reply app/controllers/stripe-setup/bank-details/post.controller.js]: Ah yes you're right 👍
